### PR TITLE
Render fidelity: upgrade Filament.

### PR DESCRIPTION
**Pull request by prideout**
This upgrade affects AlphaModeBlendTest as well as memory usage when using the test framework interactively.

### CODE REVIEW

File: packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts [hunk 1]
```diff
@@ -234,10 +244,12 @@ export class FilamentViewer extends LitElement {
...
     });
 
-    loader.delete();
-
     this[$boundingBox] = asset.getBoundingBox();
-    this[$scene].addEntities(asset.getEntities());
+    const entities = asset.getEntities();
+    this[$scene].addEntities(entities);
+    for (const entity of entities) {
+      entity.delete();
```
Comment (elalish) [inline packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts:251]: Why do we delete the entities right after adding them?

File: packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts [hunk 2]
```diff
@@ -149,37 +152,40 @@ export class FilamentViewer extends LitElement {
...
+      this[$assetLoader]!.destroyAsset(existingAsset);
       this[$currentAsset] = null;
+      for (const entity of entities) {
+        entity.delete();
+      }
     }
 
     if (this[$ibl] != null) {
       this[$scene].setIndirectLight(null);
+      this[$engine].destroyTexture(this[$ibl]!.getReflectionsTexture());
```
Comment (elalish) [inline packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts:168]: We can remove all these `!` by doing `const ibl = this[$ibl]` and then operating no `ibl` instead. Actually, I'd really rather just strip all the Symbols out of this file, but I'm not going to make you do that. :)

File: packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts (hunk 1 above)
Comment (prideout) [reply packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts:251]: This is an unfortunate emscripten-ism.  The emscripten entity wrapper is being deleted, not the entity itself.

Comment (elalish) [reply packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts:251]: ah, good to know. Is it still necessary to also delete them above then, or will they already be gone?

Comment (elalish) [reply packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts:251]: As in they are also deleted around line 160.

Comment (prideout) [reply packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts:255]: Added a commit with a comment.

File: packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts (hunk 2 above)
Comment (prideout) [reply packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts]: Added a commit to clean this up.

**Review by elalish — APPROVED** [review]
Thanks!

**Pull request merged**